lib/matobj.gi

```diff
@@ -1568,3 +1568,572 @@ InstallMethod( \[\,\]\:\=, "for a matrix object, two positions, and an object",
   end );
 
 
+############################################################################
+# Elementary matrix operations
+############################################################################
+
+############################################################################
+##
+#O  MultMatrixRow( <M>, <row>, <scalar> )
+##
+##  <#GAPDoc Label="MultMatrixRow">
+##  <ManSection>
+##  <Oper Name="MultMatrixRow" Arg='mat,i,elm' Label=""/>
+##  
+##  <Returns>nothing</Returns>
+##
+##  <Description>
+##  <P/>
+##  For a matrix <A>mat</A> such that <A>mat</A> is mutable, a row index <A>i</A>  
+##  and a scalar <A>elm</A> this operation multiplies <A>elm</A> with the <A>i</A>-th row 
+##  in-place.
+##  Note that by default <A>elm</A> is multiplied from the left. 
```

Comment:
Here you can add a cross-reference to the function that multiplies from the right.